enhancements/update/update-blocker-lifecycle/README.md

```diff
@@ -0,0 +1,93 @@
+---
+title: update-blocker-lifecycle
+authors:
+  - "@wking"
+reviewers:
+  - "@LalatenduMohanty"
+approvers:
+  - TBD
+creation-date: 2020-09-11
+last-updated: 2020-09-11
+status: implementable
+---
+
+# Update-blocker Lifecycle
+
+We occasionally have bugs which impact update success or the stability of the target release.
+When that happens, we protect users by [removing the impacted updates from our update recommendations][graph-data-block].
+This enhancement describes the process used to identify these bugs and determine which edges should be blocked because of them.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The lifecycle for update blockers looks like:
+
+<div style="text-align:center">
+  <img src="flow.svg" width="100%" />
+</div>
+
+Currently all tracking through the lifecycle is manual, and it is tedious for graph-admins to audit bugs with `UpgradeBlocker` to see where they are in the lifecycle and, when necessary, poke component teams about outstanding impact statement requests.
+Having an expliict, machine-readable lifecycle reduces the chances that suspected update blockers fall through the cracks by clarifying the responsible parties for moving the bug to the next stage, which supports tracking and automated reminders.
+
+## Motivation
+
+### Goals
+
+* Clearly define, in a machine-readable fashion, the currently responsible party for bugs in the update-blocker lifecycle.
+
+### Non-Goals
+
+* Something?
+
+## Proposal
+
+Add new `Devel Whiteboard` labels for `ImpactStatementRequested`, `ImpactStatementProposed`, and `UpdateRecommendationsBlocked`.
```

Comment:
for some reason, some people does not see `Devel Whiteboard`, including @smarterclayton I would use `Whiteboard` instead.